src/main/java/edu/ksu/canvas/requestOptions/SubmissionOptions.java

```diff
@@ -0,0 +1,124 @@
+package edu.ksu.canvas.requestOptions;
+
+import java.util.Map;
+
+/**
+ * Rubric_Assessment not supported yet.
+ */
+public class SubmissionOptions extends BaseOptions {
```

Comment:
These options classes are specific to individual API calls. Naming this class so generically will make it confusing when we implement more API calls related to submission objects.